servicetalk-concurrent-api/src/main/java/io/servicetalk/concurrent/api/Completable.java

```diff
@@ -1187,6 +1187,15 @@ public static Completable completed() {
         return CompletedCompletable.INSTANCE;
     }
 
+    /**
+     * Creates a realized completed {@code Completable} which first runs the supplied {@code Runnable}.
+     *
```

Comment:
Add an `@param` for the arguments and follow the guidance on the javadocs from `Single` above